module/zfs/spa_errlog.c

```diff
@@ -53,7 +63,17 @@
 #include <sys/spa_impl.h>
 #include <sys/zap.h>
 #include <sys/zio.h>
+#include <sys/dsl_dir.h>
+#include <sys/dmu_objset.h>
+#include <sys/dbuf.h>
 
+/*
+ * spa_upgrade_errlog_limit : A zfs module parameter that controls the number
+ * 		of on-disk error log entries that will be converted to the new
+ * 		format when enabling head_errlog. Defaults to 0 which ignores
+ * 		the setting.
+ */
+uint32_t spa_upgrade_errlog_limit = 0;
```

Comment:
`static`

Reply:
I have the impression that most other tunables are not defined as static, although they are used only in the file where they are defined.

Reply:
#12899; I don't necessarily want to spend even more hours running that on random new code.

Reply:
Great! Had not seen that. Will fix here.